relay-pii/src/eap.rs

```diff
@@ -0,0 +1,2566 @@
+//! Provides the `scrub` function for scrubbing EAP items.
+//!
+//! This module also contains tests for scrubbing attributes.
+
+use relay_event_schema::processor::{
+    ProcessValue, ProcessingResult, ProcessingState, ValueType, process_value,
+};
+use relay_protocol::Annotated;
+
+use crate::{AttributeMode, PiiConfig, PiiProcessor};
+
+/// Scrubs an EAP item (such as an `OurLog` or `SpanV2`) with the given PII configs.
+pub fn scrub<T: ProcessValue>(
+    value_type: ValueType,
+    item: &mut Annotated<T>,
+    pii_config: Option<&PiiConfig>,
+    pii_config_from_scrubbing: Option<&PiiConfig>,
```

Comment:
```suggestion
    advanced_rules: Option<&PiiConfig>,
    legacy_rules: Option<&PiiConfig>,
```

Reply:
Can't wait for #5156 and there is only a single `PiiConfig`.